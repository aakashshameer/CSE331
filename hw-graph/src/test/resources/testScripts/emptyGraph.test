#Test that creates an empty graph

#Create a graph
CreateGraph dgraph

#Print the nodes in the graph
ListNodes dgraph