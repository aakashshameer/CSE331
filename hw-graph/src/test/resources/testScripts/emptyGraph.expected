#Test that creates an empty graph

#Create a graph
create graph dgraph

#Print the nodes in the graph
dgraph contains: